packages/terra-date-picker/src/react-datepicker/year_dropdown_options.jsx

```diff
@@ -26,12 +26,33 @@ function generateYears (year, noOfYear, minDate, maxDate) {
 
 export default class YearDropdownOptions extends React.Component {
   static propTypes = {
+    /**
+     * Minimum Value of date can be selected.
+     */
     minDate: PropTypes.object,
+    /**
+     * Maximum Value of date can be selected.
+     */
     maxDate: PropTypes.object,
+    /**
+     * A callback function to execute when user clicks outside.
+     */
     onCancel: PropTypes.func.isRequired,
+    /**
+     * A callback function to execute when a valid date is selected .
+     */
     onChange: PropTypes.func.isRequired,
+    /**
+     * Prop to identify scrollable year dropdown.
```

Comment:
Can be changed to: https://github.com/cerner/terra-framework/pull/785/files#diff-0eaf044a5babfc8ff56c4b57c5220edbR243